purefa/README.md

```diff
@@ -0,0 +1,130 @@
+# Agent Check: PureFA
+
+## Overview
+
+This check monitors the [Pure Storage FlashArray][3] through the [Datadog Agent][2] and the [Pure Storage Prometheus exporter][1]. 
+
+The integration can provide performance data at the array, host, volume and pod level, as well as high-level capacity and configuration information.
+
+Monitor multiple FlashArrays and aggregate these into a single dashboard, or group together by custonmer defined environment.
+
+Note: This integration requires:
+
+ - OpenMetricsBaseCheckV2 which is available in Agent v7.26.x+
+ - Python 3
+ - The [Pure Storage Prometheus exporter][1] is installed and running in a containerized environment. Refer to the [GitHub repo][1] for installation instructions.
```

Comment:
```suggestion
 - The Pure Storage Prometheus exporter is installed and running in a containerized environment. Refer to the [GitHub repo][1] for installation instructions.
```
Small nit but both links just go to the same url, so I think removing one of them is more intuitive.

Reply:
Ack